src/EnergyPlus/DataSurfaces.cc

```diff
@@ -665,6 +685,184 @@ namespace DataSurfaces {
 			}
 		}
 
+    double SurfaceData::getInsideAirTemperature( const int t_SurfNum ) const
```

Comment:
EnergyPlus uses `Real64` instead of `double` directly due to legacy translation. You should change all instances of `double` to `Real64` instead.